### DISCUSSION TITLE
Self-Hosted Runner - You are not authorized to access https://pipelines.actions.githubuser..

### DISCUSSION THREAD

**Discussion by mjansrud** [community/community#26227] [Actions]
<p>Hello,</p>
<p>We are trying to deploy a self-hosted runner behind our firewall.<br>
What domains needs to be whitelisted for the runner to work?<br>
The application is currently running on a kubernetes cluster through a http / https proxy.</p>
<p>We have currently whitelisted<br>
<strong><a href="http://github.com" rel="nofollow noopener">github.com</a></strong><br>
<strong><a href="http://api.github.com" rel="nofollow noopener">api.github.com</a></strong><br>
<strong><a href="http://pipelines.actions.githubusercontent.com" rel="nofollow noopener">pipelines.actions.githubusercontent.com</a></strong></p>
<p>The docker-image works perfectly fine on localhost, but i get the following error on kubernetes:</p>
<pre><code>Failed to create session. You are not authorized to access https://pipelines.actions.githubusercontent.com.
</code></pre>
<p>Is it possible to activate debug-logging?<br>
Thanks in advance.</p>
<p>Log dump:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/9e71ccab9e708a4cb1b58d294af1a3e2e2b85503.png" data-download-href="/uploads/short-url/mBFfMLuk6CPLqrfQmfFzYTZZ8qf.png?dl=1" title="log.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/6075402/181096152-5f77adb1-7a1b-4d10-a520-14a2858d8f54.png" alt="log.png" data-base62-sha1="mBFfMLuk6CPLqrfQmfFzYTZZ8qf" width="601" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/9/9e71ccab9e708a4cb1b58d294af1a3e2e2b85503_2_601x500.png, https://aws1.discourse-cdn.com/github/original/2X/9/9e71ccab9e708a4cb1b58d294af1a3e2e2b85503.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/9/9e71ccab9e708a4cb1b58d294af1a3e2e2b85503.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/9e71ccab9e708a4cb1b58d294af1a3e2e2b85503_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">log.png</span><span class="informations">874×726 105 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>When you configure your self-hosted runner, which token do you use ? It is a time-limited token which will expire after one hour. </p>
<p>Now , github action provides an API to create a registration token for your self-hosted runner: </p>
<p><a href="https://developer.github.com/v3/actions/self_hosted_runners/#create-a-registration-token" rel="nofollow noopener">https://developer.github.com/v3/actions/self_hosted_runners/#create-a-registration-token</a></p>
<p>If you use a specific token now, please try to use the API to get a new token . </p>
